srcpkgs/parted/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'parted'
 pkgname=parted
-version=3.3
+version=3.4
 revision=1
 build_style=gnu-configure
 # parted wants off_t as 64bit type
 configure_args="ac_cv_sizeof_off_t=8"
```

Comment:
I really dislike this... Solving with `_LARGEFILE_SOURCE` would be much much better.